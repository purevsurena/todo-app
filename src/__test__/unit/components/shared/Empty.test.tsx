import React from "react";
import { render } from "@testing-library/react-native";
import EmptyComponent from "@/components/shared/Empty";

describe("EmptyComponent", () => {
  it("renders correctly", () => {
    const { getByText } = render(<EmptyComponent />);

    const emptyText = getByText(
      "Your todo list is empty. Add a new reminder to organize your tasks.",
    );
    expect(emptyText).toBeTruthy();
  });

  it("matches the snapshot", () => {
    const { toJSON } = render(<EmptyComponent />);
    expect(toJSON()).toMatchSnapshot();
  });
});
