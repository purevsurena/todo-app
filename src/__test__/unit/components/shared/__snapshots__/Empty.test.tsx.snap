// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyComponent matches the snapshot 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#fff",
      },
      {
        "backgroundColor": "transparent",
        "marginTop": 412.6666666666667,
        "padding": 32,
      },
    ]
  }
>
  <Text
    style={
      [
        {
          "color": "#fff",
        },
        {
          "fontSize": 16,
          "lineHeight": 24,
        },
        undefined,
        undefined,
        undefined,
        undefined,
        {
          "color": "#9E9EA5",
          "textAlign": "center",
        },
      ]
    }
  >
    Your todo list is empty. Add a new reminder to organize your tasks.
  </Text>
</View>
`;
